src/lib/mina_base/user_command.ml

```diff
@@ -326,7 +326,26 @@ let check_verifiable (t : Verifiable.t) : Valid.t Or_error.t =
   | Zkapp_command p ->
       Ok (Zkapp_command (Zkapp_command.Valid.of_verifiable p))
 
+let zero_vesting_msg = "Timing update with zero vesting period"
+
+let check_zero_vesting_period (t : t) : unit Or_error.t =
```

Comment:
Can you move this down into `Zkapp_command.Valid.of_verifiable`, so that it also gets exercised by `check_verifiable` above, and to keep a layer of abstraction between this module and the zkapp command internals?